test/spec-urls.test.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const assert = require('assert');
+const specData = require('browser-specs');
+const { visit } = require('../utils');
+
+describe('spec_url data', () => {
+  it('spec_url specifications only use allow listed hosts by w3c/browser-specs (and those we allow list for now)', () => {
+    const toVisit = [
+      'api',
+      'css',
+      'html',
+      'http',
+      'javascript',
+      'mathml',
+      'svg',
+      'webdriver',
+    ];
+
+    let specURLs = [];
+    for (const key of toVisit) {
+      visit(
+        (path, feature) => {
+          if (feature.spec_url) {
+            if (Array.isArray(feature.spec_url)) {
+              specURLs.push(...feature.spec_url);
+            } else {
+              specURLs.push(feature.spec_url);
+            }
+          }
+        },
+        {
+          entryPoint: key,
+        },
+      );
+    }
```

Comment:
Since we're not using any of the control flow that `visit()` provides, I'd probably use `walk()` instead. Something like this?

```js
const specURLs = [];
for (const { compat } of walk()) {
  const { spec_url } = compat;
  const specs = [].concat(spec_url || []); // coerce spec_url to array, or empty array if undefined
  specURLs.push(...specs);
}
```

```suggestion
  const specURLs = [];
  for (const { compat } of walk()) {
    const { spec_url } = compat;
    const specs = [].concat(spec_url || []); // coerce spec_url to array, or empty array if undefined
    specURLs.push(...specs);
  }
```